apps/extension/src/features/router/config.tsx

```diff
@@ -5,42 +5,44 @@ import {
 } from "@tanstack/react-location"
 
 import { SignIn } from "~features/auth"
-import { Calendar } from "~features/calendar"
 import { trpcApi } from "~features/trpc/api"
 
+import { Calendar } from "./routes/calendar"
 import { Settings } from "./routes/settings"
 
 // Create a memory history
 export const reactLocationOptions: ReactLocationOptions = {
   history: createMemoryHistory({
-    initialEntries: ["/authenticated"] // Pass your initial url
+    initialEntries: ["/calendar"] // Pass your initial url
```

Comment:
Or are you ignoring the `"/"` path because we have a conditional render of `SignIn` OR the `Router`?

That's what I had initially as well but saw on tanner's kitchen sink that he includes everything in the routes and does the conditional `<Outlet/>` in the `SignIn` component

<img width="483" alt="image" src="https://user-images.githubusercontent.com/18185649/199100012-7c54fecc-3681-4d6e-93ea-dffe1775e2a4.png">